items/tests.py

```diff
@@ -218,3 +218,124 @@ def test_delete_group_authorized(self):
     self.assertHttpAccepted(response)   
     groups_count = Group.objects.filter(creator=user).count()
     self.assertEqual(groups_count, 1)
+
+
+class UserResourceTest(ResourceTestCase):
+
+  def setUp(self):
+    super(UserResourceTest, self).setUp()
+    self.url = '/api/v1/user/'
```

Comment:
it's a constant, move it above like `URI = '/api/v1/user/'` and access `self.URI`.
please pay attention that url is not quite right word, uri is preferable